src/docs/guides/rocket.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Deploy a Rust Rocket App
+---
+
+[Rocket](https://rocket.rs) is a web framework for Rust that makes it simple to write fast, type-safe, secure web applications with incredible usability, productivity and performance.
+
+This guide covers how to deploy a Rocket app to Railway in four ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+Now, let's create a Rocket app! 🚀
+
+## Create a Rocket App
+
+**Note:** If you already have a Rocket app locally or on GitHub, you can skip this step and go straight to the [Deploy Rocket App to Railway](#deploy-the-rocket-app-to-railway).
+
+To create a new Rocket app, ensure that you have [Rust](https://www.rust-lang.org/tools/install) installed on your machine.
+
+Run the following command in your terminal to create a new Rust app:
+
+```bash
+cargo new helloworld --bin
+```
+
+The command creates a new binary-based Cargo project in a `helloworld` directory.
+
+Next, `cd` into the directory and add Rocket as a dependency in your `Cargo.toml`:
```

Comment:
Isn't there some cargo command that adds dependencies to the Cargo.toml file for you? similar to `go get`

Reply:
Yes there is, but you need to install the cargo-edit package first. Then you can now use `cargo add`. 

For this guide, it feels like an overkill since we are adding just `rocket` as the dependency.

Reply:
Oh wow, needing to install an extra package to be able to add packages, Rust is so amazing!

Reply:
Lmao

Reply:
No, you don't. In new rust versions cargo add is a regular command.

Reply:
Thanks, just discovered that worked. Looks I was using a really older version